modules/ocl-loader/src/main/java/org/eclipse/xpanse/modules/ocl/loader/data/models/UserData.java

```diff
@@ -16,7 +19,13 @@
 @Data
 public class UserData {
 
+    @NotBlank
+    @Schema(description = "The type of the UserData, valid values: shell, powershell")
     private UserDataType type;
+
+    @NotBlank
+    @Valid
+    @Schema(description = "The commands of the UserData")

```

Comment:
Again here NotBlank might not be valid for lists. Please make it notEmpty and also notNull